packages/onchainkit/src/swap/utils/getSwapMessage.test.ts

```diff
@@ -1,213 +1,69 @@
-import { describe, expect, it, vi } from 'vitest';
+import { describe, expect, it } from 'vitest';
```

Comment:
Hey @VolodymyrBg, this test file is for `getSwapMessage`, not `getSwapErrorCode`. Was this intentional? I don't think we wanna remove tests for `getSwapMessage`.